src/content/guides/using-persistent-volumes-on-aws-with-efs/index.md

```diff
@@ -5,6 +5,12 @@ date: "2020-02-06"
 type: page
 weight: 50
 tags: ["tutorial"]
+user_questions:
+  - How do I install the EFS provisioner?
+  - How do I provision an EFS instance on AWS?
+  - How do I test the efs storage class?
```

Comment:
```suggestion
  - How do I test the EFS storage class?
```

Reply:
The storage class is actually spelled `efs` (lowercase)

Reply:
Ok. I got confused, because the first questions had EFS in upper case, and it seemed inconsistent to me.